client/src/components/Donation/DonateForm.tsx

```diff
@@ -310,26 +309,13 @@ class DonateForm extends Component<DonateFormProps, DonateFormState> {
       isOneTime ? 'donate.confirm-2' : 'donate.confirm-3',
       { usd: donationAmount / 100 }
     )}:`;
-
-    const walletlabel = `${t(
-      isOneTime ? 'donate.wallet-label' : 'donate.wallet-label-1',
```

Comment:
If this is going, should the translation jsons be updated?

Reply:
I don't think that will cause any problems - they will just be unused keys, right? Which should be fine.

Reply:
no cause it will be coming back? would it be a problem if there is an unused key in the json?

Reply:
It's not a problem except that it's more clutter and another thing that people think they should translate. I don't think it'll break anything, though.

Reply:
If it's going to be used, it's totally fine to leave it.